src/components/CWWidget.tsx

```diff
@@ -24,7 +24,7 @@ export default class CWWidget extends Component<ICWWidgetProps> {
           />
         </View>
         <View style={this.props.style.rowContainer}>
-          <Text style={this.props.style.textDegrees}>{this.props.data.degrees}°</Text>
+          <Text style={this.props.style.textDegrees}>{this.props.data.degrees.toString()}°</Text>

```

Comment:
Please check the indentation of your files

Reply:
Mmm I think it's correctly indented, we are using Prettier and ESLint, and we don't see any issues, also Codacy didn't drop any alert.
Preview (no issues):
![image](https://user-images.githubusercontent.com/45635827/224854697-c2eff7cc-e367-4223-938a-69b853001d9c.png)

Preview (with issues):
![image](https://user-images.githubusercontent.com/45635827/224854812-58dde16c-e217-422c-8a04-dffa3b77e03d.png)

Codacy results: https://app.codacy.com/gh/BrightCoders-Institute/BCDIC22-RN-proyecto-final-team3/pullRequest?prid=11325051&bid=34206334.

Reply:
I see, in that case it looks good to me

Reply:
Thanks